linode/manager
# refactor: [M3-6893] - MUI v5 Migration - `SRC > Components > Paper`

## Description 📝

- Updates imports/exports of our `<Paper />` component to match our moden patterns

## Major Changes 🔄
- Only named export `<Paper />` (no more default export) 🧼
- Moved `<Paper />` out of the `core` folder 🗂️
- Converted Storybook story to CSF 3️⃣
- Added Storybook story for the **outlined** variant of a `<Paper />` 📄
- Added Storybook story for a `<Paper />` that has an `error` 📄

## Preview 📷
![Screenshot 2023-07-17 at 12 15 14 PM](https://github.com/linode/manager/assets/115251059/eb6c4183-2bec-469a-ad44-63f70d5ee9b9)

## How to test 🧪
- Verify automated testing and type-checking passes
- Check over the app for visual regressions
